core-web/libs/sdk/react/src/lib/next/components/Row/Row.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from './Row.module.css';
+
+import { DotPageAssetLayoutRow } from '../../types';
+import { Column } from '../Column/Column';
+
+type RowProps = {
+    row: DotPageAssetLayoutRow;
+};
+/**
+ * This component renders a row with all it's content using the layout provided by dotCMS Page API.
+ *
+ * @see {@link https://www.dotcms.com/docs/latest/page-rest-api-layout-as-a-service-laas}
+ * @category Components
+ * @param {React.ForwardedRef<HTMLDivElement, RowProps>} ref
+ * @return {JSX.Element} Rendered rows with columns
+ */
+export const Row = ({ row }: RowProps) => {
+    const rowProps = { 'data-dot': 'row' };
+
+    return (
+        <div className={row.styleClass}>
+            <div className="container">
```

Comment:
should we move already to `dot-container`

Reply:
Sure thing!